### ISSUE TITLE
Table view with details

### ISSUE THREAD

**Issue by alexandernst** [yaronn/blessed-contrib#36]
It would be nice if you could implement a detailed table view, like this one: http://www.zuschlogin.com/content/blogimages/31/Telescope.gif

Maybe not that overloaded, but make it possible to place a `+` button at the left side of  each row, and when clicked, expand the row and show some kind of text area.

**Comment by yaronn** [comment]
@theredcat - you have been doing some work around this, do you have some preview to share?

**Comment by theredcat** [comment]
I'm doing more testing and adding examples, but the plugin should work right now.

@alexandernst you can execute the default dashboard of my fork to see the widget preview : https://github.com/theredcat/blessed-contrib . I'll be happy to receive suggestions about features

**Comment by alexandernst** [comment]
@theredcat Ok, I will test it tonight (in a few hours) and let you know if I have any suggestions :)

**Comment by alexandernst** [comment]
@theredcat I tried your master brach, but I'm not sure your process tree example is what I'm  asking for here. Can your tree widget act as a table?

**Comment by theredcat** [comment]
@alexandernst Indeed, no. I'm using Blessed lists to display tree so every line is just a string

**Comment by alexandernst** [comment]
@theredcat Do you think it would be difficult to implement a table view with details?

**Comment by theredcat** [comment]
@alexandernst I've actually though about it. But this mean I can't use the List element anymore and each individual line must be a Box. I may try to integrate this in the futur, but this won't be in my next MR

**Comment by alexandernst** [comment]
@theredcat Can you ping  me here if/when you have something  I can test, please?

**Comment by theredcat** [comment]
@alexandernst Of course :)

**Comment by alexandernst** [comment]
@theredcat Any news on this one? :)
